articles/mathematics-for-machine-learning/index.md

```diff
@@ -0,0 +1,162 @@
+### Prerequisites
+
+The equivalent of high school maths is needed to understand the concepts used in Machine Learning (ML), such as linear algebra, probability and statistics, and multivariate calculus. Yet, a background in ML is necessary to understand the ML concepts and algorithms. 
+
+### Introduction
+
+Is it necessary to understand the mathematics behind ML?
+
+Absolutely. 
+
+Machine learning is all mathematics. ML is built on mathematical prerequisites. The math helps you understand why some models are better than others.
+
+The chart below shows the importance of each mathematical concept needed to master ML.
+
+![The chart shows the importance of each mathematical concept](/engineering-education/mathematics-for-machine-learning/math-topics.png)<br>
+
+It can be summarized below as:
+
+1. Linear Algebra (35.0%)
+2. Probability and Statistics (25.0%)
+3. Calculus (15.0%)
+4. Algorithms (15.0%)
+5. Others (10.0%)
+
+This article is not intended to go into details about the mathematical calculations in the various topics. Instead, it's meant to give you a general overview of the math topics behind ML and how they are used. 
+
+Let's dive into it.
+
+### Linear Algebra
+
+Linear Algebra covers a significant part of the mathematical concepts used for Machine Learning. According to [Wikipedia](https://en.wikipedia.org/wiki/Linear_algebra/), Linear Algebra is a mathematics domain concerned with linear equations and the way they are represented in vector spaces and through matrices.
+
+The topics in Linear Algebra essential for understanding the methods used in machine learning include:
```

Comment:
Add links to resources to each of these topics. See this as a list of resources. So adding links to each of these resources will enhance the value delivered. Give a link for each subtopic under each subject.